templates/ingress/ingress-ui.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.ui.create -}}
```

Comment:
Change filename to UI, remember the API also

Reply:
Done